Winget-AutoUpdate/functions/Update-App.ps1

```diff
@@ -30,12 +30,12 @@ Function Update-App ($app) {
 
     #Run Winget Upgrade command
     if ($ModsOverride) {
-        Write-ToLog "-> Running (overriding default): Winget upgrade --id $($app.Id) --accept-package-agreements --accept-source-agreements --override $ModsOverride"
-        & $Winget upgrade --id $($app.Id) --accept-package-agreements --accept-source-agreements --override $ModsOverride | Where-Object { $_ -notlike "   *" } | Tee-Object -file $LogFile -Append
+        Write-ToLog "-> Running (overriding default): Winget upgrade --id $($app.Id) -e --accept-package-agreements --accept-source-agreements -s winget -h --override $ModsOverride"
+        & $Winget upgrade --id $($app.Id) -e --accept-package-agreements --accept-source-agreements -s winget -h --override $ModsOverride | Where-Object { $_ -notlike "   *" } | Tee-Object -file $LogFile -Append
```

Comment:
`-h` should be set even for override? Can be in conflict with parameters I think, in case someone want full interactive mode (`-i` if i remeber)

Reply:
Thanks, fixed